package.json

```diff
@@ -8,15 +8,22 @@
     "name": "Kandha Vivek Raj",
     "email": "kandhavivekraj@gmail.com"
   },
+  "bin": {
+    "git-changed-files": "lib/cli.js"
```

Comment:
we can use "bin": "lib/cli.js" directly da

Reply:
Yes, We can. But this will not be a problem. Anyways i'll change it